src/common/SPELLS/priest.ts

```diff
@@ -48,6 +48,12 @@ const spells = spellIndexableList({
     icon: 'spell_holy_mindsooth',
     manaCost: 500,
   },
+  MINDGAMES_TALENT: {
+    id: 375901,
+    name: 'Mindgames',
+    icon: 'ability_revendreth_priest',
+    manaCost: 1000,
+  },
```

Comment:
don't add talents here. they're auto-generated in `src/common/TALENTS/priest.ts`. if the spec's talents are out of date, you can run the generate-talents script or ask in discord and someone can regenerate them for you :)